frame/contracts/src/lib.rs

```diff
@@ -321,10 +321,6 @@ pub mod pallet {
 			Storage::<T>::process_deletion_queue_batch(weight_limit)
 				.saturating_add(T::WeightInfo::on_initialize())
 		}
-
```

Comment:
@athei please confirm this is not used.

Reply:
How would I know if anyone uses this? It will migrate when appropriate. The code isn't there for fun.

Reply:
well this is exactly the reason why we are removing these hooks, because it is not clear who and when uses them. 

My rephrased question is: are there any teams that you are aware of that would be affected if we remove this code? 

note that I will re-expose this migration code as a standalone function, in case anyone needs it so they can manually apply it, and mark the PR is `release-note`.

Reply:
Its fine then. But I really much preferred the old way where it just worked. Won't I get all these dead code warnings after this change?

Reply:
You will see the outcome soon.